docs/topics/native/native-objc-interop.md

```diff
@@ -173,8 +173,31 @@ with an additional receiver parameter. These types include:
 
 Kotlin singleton (made with an `object` declaration, including `companion object`)
 is imported to Swift/Objective-C as a class with a single instance.
-The instance is available through the factory method, i.e. as
-`[MySingleton mySingleton]` in Objective-C and `MySingleton()` in Swift.
+
+The instance is available through the `shared` and `companion` properties.
```

Comment:
The same should be reflected in the table at the beginning of the document.

Reply:
I've updated the table,
please, take a look